olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -76,7 +76,7 @@ spec:
   customresourcedefinitions:
     owned:
     - description: Represents an installation of a particular version of Knative Serving
-      displayName: Knative Serving
+      displayName: Knative Serving 0.11.1
```

Comment:
Should this just stay "Knative Serving"?

Reply:
I don't know. I was trying to include what I personally would expect to see if we have to show two of them. If I show the version of one, I know that would trigger in myself to know what the version of the other one is.

Reply:
So as discussed in Slack yesterday, do we want to make this "Knative Serving" and the other "Knative Serving (deprecated)"? I feel like that sufficiently conveys that we want people to move to the "proper" one while the other is going away in the future.

Reply:
Having the version there is confusing, IMO. I suppose people will think they can choose between https://github.com/knative/serving/releases/tag/v0.11.1 and https://github.com/knative/serving/releases/tag/v0.10.0 but they won't get Knative Serving of the given version, they will only get KnativeServing CRD of that version. And the version of Knative Serving won't be affected by the choice.